### PR TITLE
x11-misc/dunst: version bump to 1.6.1

### PR DESCRIPTION
**Pull request by epsilon-0**
new maintainer

Package-Manager: Portage-3.0.16, Repoman-3.0.2
Signed-off-by: Aisha Tammy <gentoo@aisha.cc>

### CODE REVIEW

**Review by titanofold — COMMENTED** [review]
According to upstream, Wayland is optional. Is this not the case?

https://github.com/dunst-project/dunst#dependencies

**Comment by epsilon-0** [comment]
> According to upstream, Wayland is optional. Is this not the case?
> 
> https://github.com/dunst-project/dunst#dependencies

Yes, you are correct. I've made it optional now.

**Comment by titanofold** [comment]
> > According to upstream, Wayland is optional. Is this not the case?
> > https://github.com/dunst-project/dunst#dependencies
> 
> Yes, you are correct. I've made it optional now.

Great, thanks!

Lastly, have you contacted desktop-misc@ to see if they're up for being removed?

**Review by titanofold — APPROVED** [review]

File: x11-misc/dunst/dunst-1.6.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit systemd toolchain-funcs
```
Comment (juippis) [inline x11-misc/dunst/dunst-1.6.1.ebuild:5]: We like to have an empty line below EAPI.
https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide/Style_Guide#One_empty_line_below_EAPI.3D_line

**Review by juippis — APPROVED** [review]

File: x11-misc/dunst/dunst-1.6.1.ebuild (hunk 1 above)
Comment (epsilon-0) [reply x11-misc/dunst/dunst-1.6.1.ebuild:5]: :P oops, sorry seems I forgot that, will be more careful

**Pull request closed without merging**